### ISSUE TITLE
Could not find constructor in registry for MDCTextFieldHelperText

### ISSUE THREAD

**Issue by devvercer** [material-components/material-components-web#2975]
It seems MDCTextFieldHelperText is not registered for auto init.

**Comment by kfranqueiro** [comment]
That's correct - the Text Field subcomponents are not registered; only the top-level text field component is registered. The text field component takes care of instantiating its subcomponents.

Is there a reason you want to auto-init just the helper text?

**Comment by devvercer** [comment]
No, just that I assumed that all components that were documented to call
their constructors should do so.

https://material.io/develop/web/components/input-controls/text-field/helper-text/

**Comment by moog16** [comment]
If you're using the text-field component there is no need to initialize your own helper-text component, which is why we left it out of the subcomponents. 

Or are you building your own text-field and using our helper-text component?

**Comment by devvercer** [comment]
No it is working fine as far as I can tell!  Might be worth documenting
that MDCTextField also creates the objects for its subcomponents.
